trie/database.go

```diff
@@ -257,6 +258,17 @@ func expandNode(hash hashNode, n node) node {
 	}
 }
 
+// trienodeHasher is a struct to be used with BigCache, which uses a Hasher to determine which shard to place an
+// entry into. It's not a cryptographic hash, just to provide a bit of anti-collision.
+// By default, FNV64a is used
+// Since trie keys are already hashes, we can just use the key directly to map shard id
```

Comment:
Nitpick, but could you please justify this (and your other comments in other PRs ;P) to +-80 characters. It looks so nasty that the comment is totally randomly jagged.

Reply:
Also, please use full sentences for docs (i.e. period at the end).

Reply:
fixed